app/assets/javascripts/constants/image_constants.es6.jsx.erb

```diff
@@ -1,11 +1,19 @@
 (() => {
   class ImageConstants {
-    
+
     get logos() {
       return {
         orange: "<%= asset_path("logo/logo_orange.png") %>"
       };
     }
+
+    get roles() {
+      return {
+        donor: "<%= asset_path("logo/logo_orange.png") %>",
+        teacher: "<%= asset_path("role/teacher.png") %>"
```

Comment:
@sonia-y right now when I use this, it doesn't render: 
![image](https://cloud.githubusercontent.com/assets/5432430/18986249/e8f82304-86b0-11e6-9cc5-cb3edf5381e0.png)

Not sure what i'm missing....